packages/core/src/index.ts

```diff
@@ -93,17 +94,33 @@ export const create = (config: Config): VoiceCompass => {
       return;
     }
 
+    const _payload = {
```

Comment:
What do you mean to imply with the `_` prefix in the naming? Can we think of a way to make the name indicate this intention explicitly? These underscores have lost quite a bit of semantic meaning for me, maybe you see this differently?

Reply:
how about payload?

Reply:
i'm misusing the underscore convention, it was a way to indicate private / non-exported variables which doesn't even make sense thinking about it bc its being passed into an http call. soo yeah. i wont use it going forward and will think of a better name

Reply:
`payload` makes sense to me :)